main.py

```diff
@@ -342,8 +303,12 @@ def main():
     parser.add_argument('-v', '--version', help="The version of this software.",
         dest="version", action="store_true", required=False)
 
-    parser.add_argument('-c', '--command', help="This will run a command on " \
-        "the Minecraft Server", dest="command", type=str, required=False)
+    parser.add_argument('-c', '--command', help="If ran by itself, without " \
+        "the addition of an argument, this will start a terminal which you " \
+        "can enter multiple commands in sequence. If an argument is passed " \
+        "it will run that command only. In order to stop the terminal you " \
+        "can enter '!exit' and it will safely end the terminal session.", 
+        dest="command", nargs="?", const="", type=str, required=False)
```

Comment:
We certainly could. I mean, I'm definitely open to the idea. I just don't want to bog the help menu, and add so many different arguments where we might be fighting to make clean-easy-to-understand acronyms for the arguments. You know? But yeah, I don't have a problem with that if you think it would be easier for the user.

Reply:
Honestly, I just don't like how much space this one command takes up in the help menu now. I know there are other arguments from other applications that take up a LOT more, so I'm okay either way. The thought popped into my head while reviewing, so I thought we'd talk about it. 

If we break it up, we can simply say something like "Issue a command or list of commands to your Minecraft server." for -c and something like "Start a command interface to issue multiple commands to your Minecraft server. You can enter '!exit' at any time to safely end the session."

Reply:
Hmmm... I guess thats a decision we need to make amongst ourselves. Which is the greater evil, longer help entries or more arguments?

Reply:
We could look look at subparsing to get the best of both worlds? Maybe the command could be `python main.py -c interface`?

I think I'm going to add subparsing the maintenance parser to 1.5, actually.

Reply:
https://stackoverflow.com/questions/46719811/best-practices-for-writing-argparse-parsers

Reply:
@BAChapin I'm going to resolve this conversation so we can move forward for now, but I do think that it might be wise to take a serious look at arguments in the 1.5 refactor.